Library/Homebrew/test/tab_test.rb

```diff
@@ -241,15 +241,18 @@ def test_to_json
     assert_equal @tab.changed_files, tab.changed_files
     assert_equal @tab.tap, tab.tap
     assert_equal @tab.spec, tab.spec
-    assert_equal @tab.time, tab.time
+    assert_nil @tab.time
+    assert_nil tab.time
```

Comment:
@reitermarkus MiniTest whines if you check for equality as we did before and warns it'll break in a later version so this will need to stay as-is for now.